app/src/main/java/fr/free/nrw/commons/upload/PendingUploadsFragment.kt

```diff
@@ -140,19 +141,24 @@ class PendingUploadsFragment :
     /**
      * Restarts all the paused uploads.
      */
-    fun restartUploads() = pendingUploadsPresenter.restartUploads(
-        contributionsList, 0, requireContext().applicationContext
-    )
+    fun restartUploads() {
```

Comment:
My understanding was that we don't need any changes in this entire file, is there anything that I'm missing out?😕